docs/guides/ros2-gem.md

```diff
@@ -88,6 +90,15 @@ You can use tools such as [rqt_robot_steering](https://index.ros.org/p/rqt_robot
 
 It is possible to implement your own control mechanisms with this Component.
 
+#### Spawner
+
+`ROS2SpawnerComponent` handles spawning entities during simulation.
+Available spawnables have to be set up as the component's field before the simulation.
+
+During the simulation user can access names of available spawnables and request spawning using ros2 services. GetWorldProperties.srv and SpawnEntity.srv types from gazebo_msgs are used to handle these features.
```

Comment:
A bit more information here - you can refer to a design document as well. Which topics, how to access.